src/components/mol.menu/menu.jsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react'
-import { NavContainer, Overlay } from './menu.styled';
+import { NavContainer, NavListContainer, Overlay } from './menu.styled';
 import { HeaderNav } from '../mol.header/header';
+import { H3 } from '../atm.typography/typography.styled';
+import { HBox } from '../mol.box/box.styled';
+import { Link } from 'react-router-dom';
 
 export const Menu = props => {
     return (
         <>
             <NavContainer opened={props.opened}>
                 <HeaderNav children={'Pedro'} handleCloseClick={props.handleClick} />
+                <NavListContainer>
+                    <Link to={'/'}>
```

Comment:
quando começar a surgir mais rotas (e vai pelo design), pode ter mta repeticao de Link -> Hbox -> H3.
Pode tentar melhorar essa parte? pode tentar criar todos links q ta no design, msm q n tenha componente Route implementado pra ele.

Reply:
Se for implementar um array de objetos e um map eu fiz isso.